torch_geometric/data/data.py

```diff
@@ -842,6 +842,9 @@ def _put_edge_index(self, edge_index: EdgeTensorType,
         attr_val = edge_tensor_type_to_adj_type(edge_attr, edge_index)
         setattr(self, attr_name, attr_val)
 
+        # Set edge attributes:
```

Comment:
Won't we need to have special logic in `_get_edge_index` as well?

Reply:
What kind of special logic? `get_edge_index` just returns the edge corresponding to the requested layout. If we want to handle `is_sorted`, we can handle it at a higher level in `GraphStore`, but I left a TODO to do that later - wdyt?

Reply:
Got it.